R/pkg/R/RDD.R

```diff
@@ -97,7 +97,8 @@ setMethod("initialize", "PipelinedRDD", function(.Object, prev, func, jrdd_val)
     # prev_serializedMode is used during the delayed computation of JRDD in getJRDD
   } else {
     pipelinedFunc <- function(partIndex, part) {
-      func(partIndex, prev@func(partIndex, part))
+      f <- prev@func
```

Comment:
Because a warning about a space before left parenthesis was caught. I tried to do as follows. But lintr caught a warning about that.

```
func(partIndex, (prev@func)(partIndex, part))
```